css/css-grid/masonry/tentative/intrinsic-sizing/masonry-intrinsic-sizing-cols-001.html

```diff
@@ -230,16 +233,93 @@
   <item>3</item>
   <item>4</item>
   <item style="grid-column:2/span 2">5</item>
+  <item style="width:5ch; grid-column:span 3/4">6</item>
```

Comment:
Overall comment, I appreciate you breaking these up into sections when viewing visually. Having some text to identify which section we are looking at would be nice.